importlib_metadata/_compat.py

```diff
@@ -52,5 +51,7 @@ def pypy_partial(val):
 
     Workaround for #327.
     """
+    import platform
```

Comment:
I'm slightly inclined to say we shouldn't bother moving `platform`. This import only happens in `_compat`, which only happens in zipp (and not in zipfile). Especially given that importing `platform` unconditionally only contributes ~2% to the startup time, I believe the maintenance costs outweigh the benefits. I'll back out this change.

Reply:
Done in 18eb2da0ee267394c1735bec5b1d9f2b0fa77dd9.